kvserver/kvadmission: add bulk low pri that is always subject to elastic

dt opened the pull request:
This new priority is above TTL but below bulk normal and user low. All KV requests at or below this new level are subjected to elastic limiting.

Release note: none.

Epic: none.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/129564)
<!-- Reviewable:end -->

dt committed:
admission: rename ttl-low-pri to bulk-low-pri

Release note (ops change): The lowest admission control priority for the storage layer has
been renamed from 'ttl-low-pri' to 'bulk-low-pri' in a handful of metrics.

Epic: none.

dt committed:
crosscluster/logical: use bulk-low-pri for LDR

Release note: none.
Epic: none.

dt committed:
admission: elastic limit all bulk-low-pri requests

Previously bulk low pri (formerly called ttl low pri) requests were only subject
to elastic limiting if they were also read-only requests. However writing
requests such as those performed by LDR also benefit from elastic limiting.

Release note: none.
Epic: none.

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 5 of 5 files at r1, 1 of 1 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/129564#-:-O5J6Ck86bA2faKqJOkk:bg7igkv)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @dt and @stevendanna)

___
*[`pkg/util/admission/admissionpb/admissionpb.go` line 30 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/129564#-O5J5oq35bS1QvprU83E:-O5J5oq35bS1QvprU83F:b-t0u7fm) ([raw file](https://github.com/cockroachdb/cockroach/blob/b87b82f6122f4e63b9070e3940e7de65efe201c8/pkg/util/admission/admissionpb/admissionpb.go#L30)):*
> ```Go
> 	// LowPri is low priority work.
> 	LowPri WorkPriority = math.MinInt8
> 	// BulkLowPri is low priority work from TTL internal submissions.
> ```

nit: comment is stale


<!-- Sent from Reviewable.io -->

dt commented:
TFTR!

bors r+
